frontend/src/scenes/session-recordings/player/PlayerMeta.tsx

```diff
@@ -120,6 +130,25 @@ export function PlayerMeta(): JSX.Element {
         )
     }
 
+    const windowOptions: LemonSelectOption<string | null>[] = [
+        {
+            label: <IconWindow value={currentWindowIndex} className="text-muted-alt" />,
+            value: null,
+            labelInMenu: <>Follow the user</>,
```

Comment:
It looked during the gif like this option also had an `IconWindow` in it. That somewhat confused me but I'm guessing it has changed

Reply:
yes, i realised it didn't make sense :)